Sources/X509/CMakeLists.txt

```diff
@@ -13,105 +13,7 @@
-  "CryptographicMessageSyntax/CMSSignerInfo.swift"
-  "CryptographicMessageSyntax/CMSVersion.swift"
-  "Digests.swift"
-  "DistinguishedName.swift"
-  "DistinguishedNameBuilder/CommonName.swift"
-  "DistinguishedNameBuilder/CountryName.swift"
-  "DistinguishedNameBuilder/DNBuilder.swift"
-  "DistinguishedNameBuilder/LocalityName.swift"
-  "DistinguishedNameBuilder/OrganizationName.swift"
-  "DistinguishedNameBuilder/OrganizationalUnitName.swift"
-  "DistinguishedNameBuilder/StateOrProvinceName.swift"
-  "DistinguishedNameBuilder/StreetAddress.swift"
-  "Error.swift"
-  "Extension Types/AuthorityInformationAccess.swift"
-  "Extension Types/AuthorityKeyIdentifier.swift"
-  "Extension Types/BasicConstraints.swift"
-  "Extension Types/ExtendedKeyUsage.swift"
-  "Extension Types/ExtensionIdentifiers.swift"
-  "Extension Types/KeyUsage.swift"
-  "Extension Types/NameConstraints.swift"
-  "Extension Types/SubjectAlternativeName.swift"
-  "Extension Types/SubjectKeyIdentifier.swift"
-  "Extension.swift"
-  "Extensions.swift"
-  "ExtensionsBuilder.swift"
-  "GeneralName.swift"
-  "LockedValueBox.swift"
-  "OCSP/BasicOCSPResponse.swift"
-  "OCSP/DirectoryString.swift"
-  "OCSP/OCSPCertID.swift"
-  "OCSP/OCSPCertStatus.swift"
-  "OCSP/OCSPExtensionID.swift"
-  "OCSP/OCSPNonce.swift"
-  "OCSP/OCSPPolicy.swift"
-  "OCSP/OCSPRequest.swift"
-  "OCSP/OCSPResponse.swift"
-  "OCSP/OCSPResponseBytes.swift"
-  "OCSP/OCSPResponseData.swift"
-  "OCSP/OCSPResponseStatus.swift"
-  "OCSP/OCSPSignature.swift"
-  "OCSP/OCSPSingleRequest.swift"
-  "OCSP/OCSPSingleResponse.swift"
-  "OCSP/OCSPTBSRequest.swift"
-  "OCSP/OCSPVersion.swift"
-  "PKCS8PrivateKey.swift"
-  "PromiseAndFuture.swift"
-  "RDNAttribute.swift"
-  "RandomNumberGenerator+bytes.swift"
-  "RelativeDistinguishedName.swift"
-  "SEC1PrivateKey.swift"
-  "Signature.swift"
-  "SignatureAlgorithm.swift"
-  "Verifier/AllOfPolicies.swift"
-  "Verifier/AnyPolicy.swift"
-  "Verifier/CertificateStore.swift"
-  "Verifier/OneOfPolicies.swift"
-  "Verifier/PolicyBuilder.swift"
-  "Verifier/RFC5280/BasicConstraintsPolicy.swift"
-  "Verifier/RFC5280/DNSNames.swift"
-  "Verifier/RFC5280/DirectoryNames.swift"
-  "Verifier/RFC5280/ExpiryPolicy.swift"
-  "Verifier/RFC5280/IPConstraints.swift"
-  "Verifier/RFC5280/NameConstraintsPolicy.swift"
-  "Verifier/RFC5280/RFC5280Policy.swift"
-  "Verifier/RFC5280/URIConstraints.swift"
-  "Verifier/RFC5280/VersionPolicy.swift"
-  "Verifier/ServerIdentityPolicy.swift"
-  "Verifier/TrustRootLoading.swift"
-  "Verifier/UnverifiedChain.swift"
-  "Verifier/VerificationDiagnostic.swift"
-  "Verifier/Verifier.swift"
-  "Verifier/VerifierPolicy.swift"
-  "X509BaseTypes/AlgorithmIdentifier.swift"
-  "X509BaseTypes/ECDSASignature.swift"
-  "X509BaseTypes/RSAPKCS1PublicKey.swift"
-  "X509BaseTypes/SubjectPublicKeyInfo.swift"
-  "X509BaseTypes/TBSCertificate.swift"
-  "X509BaseTypes/Time.swift"
-  "X509BaseTypes/TimeCalculations.swift"
-  "X509BaseTypes/Validity.swift")
+)
```

Comment:
These changes look wrong, want to re-run the script?

Reply:
Thanks for catching this. I was missing `gfind` via `brew install findutils` that was causing the script to fail.